test/Nockma/Compile/Positive.hs

```diff
@@ -167,7 +172,7 @@ eqStack st = eqSubStack st []
 unfoldTerm :: Term Natural -> NonEmpty (Term Natural)
 unfoldTerm t = case t of
   TermAtom {} -> t :| []
-  TermCell Cell {..} -> _cellLeft NonEmpty.<| unfoldTerm _cellRight
+  TermCell (Cell l r) -> l NonEmpty.<| unfoldTerm r
```

Comment:
Why this change? Doesn't it go against our general style (of using named fields)?